cogs/help.py

```diff
@@ -245,12 +238,6 @@ async def interaction_check(self, interaction: Interaction) -> bool:
         if interaction.channel.archived or interaction.channel.locked:  # type: ignore
             return False
 
-        if (
-            isinstance(interaction.user, Member)
-            and interaction.user.timeout is not None
-        ):
-            await interaction.send(timeout_message, ephemeral=True)
```

Comment:
Is timeout_message used anywhere else? Maybe the variable can be removed?

Reply:
yeah, it's no logner used, removed the variable!